src/backend/main/java/com/example/boilerscout/api/ProfileController.java

```diff
@@ -109,23 +109,26 @@
     }
 
 
-    public Map<String, Object> getProfile(String userId, String token, String goal) {
+    public Map<String, Object> getProfile(String userId, String token, String query) {
         Map<String, Object> response = new HashMap<String, Object>();
+        response.put("token",token);
+        response.put("userId",userId);
```

Comment:
you're putting the `token` and `userId` into the response in two separate places...only needs to be done in one place (why not at the end?)